lib/vmdb/fast_gettext_helper.rb

```diff
@@ -65,8 +65,7 @@ def self.register_locales
                                   :path           => locale_path,
                                   :type           => :po,
                                   :report_warning => false)
-
-      FastGettext.default_available_locales = find_available_locales
+      FastGettext.available_locales = find_available_locales
```

Comment:
AFAICT it behaved correctly on thin....But I can scrap this merge revert and just revert out the puma commit instead?